### PR TITLE
[northwood-studios/SCPSL-Translations] New pull request : Changing and Updating Hungarian

### PR DESCRIPTION
**Pull request by omgiamhungarian**
# SCP:SL Translation PR Template

## Language
Hungarian

## Type of change
- [ ] Added new language translation
- [x] Changed a currently existing translation

## Have you tested the changes you've made?
- [x] Yes
- [ ] No

# Checklist:
- [x] I have made sure to not commit changes to other stuff than my own or the default english one
- [ ] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser)
- [x] I have made sure that I'm not submitting a duplicate translation 
- [x] I have not included any advertisements/pointers to non-game stuff other than author in the folder name
- [ ] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of new translations

### CODE REVIEW

**Comment by Rnen** [comment]
image

**Comment by omgiamhungarian** [comment]
Sorry, I didn't notice that.
Edited.

File: Translations/hu/Death_Causes.txt [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+Masszív mennyiségű szövetelhalás van jelen a testen.
+Az életjelek hirtelen megszűntek. Nincs tisztázott sérülés.
+A halál oka ismeretlen .
+Ez <b>[user]</b> teste - a következő volt: [class]!\n\nA halál oka: [cause]
+Masszív sérülések és túl nagy mennyiségű vérveszteség.
+Nagy, mély harapásnyomok lelhetőek fel a koponya körül.
+Mállás rendkívül korrózív anyag által.
+Apró szilánkok és harmadfokú égések egy robbanás által.
+Sápadt arc, illetve a szájból kifolyó vér belső vérzésre utal, egy anyag emésztése okozhatta.
+Újra elzárva egy ember által.
```
Comment (Dat-Joseph) [inline Translations/hu/Death_Causes.txt]: This file isnt up to date. A line is currently missing. Please fix that :)

File: Translations/hu/Facility.txt [hunk 2]
```diff
@@ -17,7 +17,7 @@ EGÉR ÉRZÉKENYSÉG
 FELTÖRÉS
 FELVÉTEL...
 A fényerő emelése <b>jelentősen rontja a hangulatát </b>a játéknak.\nPróbáld olyan alacsonyan tartani amennyire csak lehet.\n<b>Az eredeti érték ajánlott.</b>
-[1] AJTÓ NYITÁSA/ZÁRÁSA\n[2] ZÁRT AJTÓ*\n\n*Az ajtó zárása a speciális képességgel történik
+{0} AJTÓ NYITÁSA/ZÁRÁSA\n{1} ZÁRT AJTÓ*\n\n*Az ajtó zárása a speciális képességgel történik
```
Comment (Dat-Joseph) [inline Translations/hu/Facility.txt]: Do not change the brackets to the swung brackets ( {} ). 
Always refer to the english default.

**Pull request merged**